### PR TITLE
Actualize ConsumerTopic, KafkaConsumer, KafkaConsumerTopic, Message

### PR DESCRIPTION
**Pull request by Tony-Sol**
Actualize ConsumerTopic, KafkaConsumer, KafkaConsumerTopic, Message due to original stubs from https://github.com/arnaud-lb/php-rdkafka and doc from https://arnaud.le-blanc.net/php-rdkafka-doc/phpdoc/book.rdkafka.html

### CODE REVIEW

File: stubs/RdKafka/KafkaConsumer.php [hunk 1]
```diff
@@ -79,6 +79,16 @@ public function getSubscription()
     {
     }
 
+    /**
+     * @param string    $topic_name
+     * @param TopicConf $topic_conf
+     *
+     * @return Topic
```
Comment (kwn) [inline stubs/RdKafka/KafkaConsumer.php:86]: I'm wondering if we should be more specific here. The docs say:

"Returns a [RdKafka\ConsumerTopic](https://arnaud.le-blanc.net/php-rdkafka-doc/phpdoc/class.rdkafka-consumertopic.html)."

File: stubs/RdKafka/KafkaConsumer.php [hunk 2]
```diff
@@ -144,4 +154,20 @@ public function getOffsetPositions($topics)
     public function close()
     {
     }
+
+    /**
+     * @param TopicPartition[] $topic_partitions
+     * @return TopicPartition[]
+     */
+    public function pausePartitions($topic_partitions)
```
Comment (kwn) [inline stubs/RdKafka/KafkaConsumer.php:162]: I couldn't find it in the docs: 
https://arnaud.le-blanc.net/php-rdkafka-doc/phpdoc/class.rdkafka-kafkaconsumer.html

Is that something that was introduced recently and the official docs hasn't been updated yet?

File: stubs/RdKafka/KafkaConsumer.php [hunk 3]
```diff
@@ -144,4 +154,20 @@ public function getOffsetPositions($topics)
...
+    {
+    }
+
+    /**
+     * @param TopicPartition[] $topic_partitions
+     * @return TopicPartition[]
+     */
+    public function resumePartitions($topic_partitions)
```
Comment (kwn) [inline stubs/RdKafka/KafkaConsumer.php:170]: Same as above.

**Review by kwn — COMMENTED** [review]
Thanks for an update. Just left a few comments. Could you please address them in a spare time?

File: stubs/RdKafka/KafkaConsumer.php (hunk 2 above)
Comment (Tony-Sol) [reply stubs/RdKafka/KafkaConsumer.php:162]: @kwn yep, it is in stub from base repo - https://github.com/arnaud-lb/php-rdkafka/blob/6.x/kafka_consumer.stub.php

File: stubs/RdKafka/KafkaConsumer.php (hunk 3 above)
Comment (Tony-Sol) [reply stubs/RdKafka/KafkaConsumer.php:170]: @kwn same - from https://github.com/arnaud-lb/php-rdkafka/blob/6.x/kafka_consumer.stub.php

File: stubs/RdKafka/KafkaConsumer.php (hunk 1 above)
Comment (Tony-Sol) [reply stubs/RdKafka/KafkaConsumer.php:86]: @kwn yep, thats my mistake, it should by KafkaConsumerTopic according to this https://github.com/arnaud-lb/php-rdkafka/blob/6.x/kafka_consumer.stub.php#L52

**Review by kwn — APPROVED** [review]

**Comment by kwn** [comment]
Thanks @Tony-Sol. Released in 2.2.1.

**Pull request merged**